server/src/main/kotlin/com/kamelia/hedera/mail/MailService.kt

```diff
@@ -0,0 +1,96 @@
+package com.kamelia.hedera.mail
+
+import com.kamelia.hedera.core.MailSendingException
+import com.kamelia.hedera.core.PasswordResetMessagingException
+import com.kamelia.hedera.plugins.toHTML
+import com.kamelia.hedera.rest.setting.Locale
+import com.kamelia.hedera.rest.user.User
+import com.kamelia.hedera.util.Environment
+import com.kamelia.hedera.util.Environment.mailFrom
+import com.kamelia.hedera.util.Environment.mailFromName
+import com.kamelia.hedera.util.Environment.mailHost
+import com.kamelia.hedera.util.Environment.mailPassword
+import com.kamelia.hedera.util.Environment.mailPort
+import com.kamelia.hedera.util.Environment.mailUseAuth
+import com.kamelia.hedera.util.Environment.mailUseTLS
+import com.kamelia.hedera.util.Environment.mailUsername
+import io.ktor.server.freemarker.*
+import jakarta.mail.Authenticator
+import jakarta.mail.MessagingException
+import jakarta.mail.PasswordAuthentication
+import jakarta.mail.Session
+import jakarta.mail.Transport
+import jakarta.mail.internet.MimeBodyPart
+import jakarta.mail.internet.MimeMessage
+import jakarta.mail.internet.MimeMultipart
+import java.util.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+object MailService {
+
+    private val props = Properties().apply {
+        this["mail.smtp.host"] = mailHost
+        this["mail.smtp.port"] = mailPort
+        this["mail.smtp.auth"] = mailUseAuth
+        this["mail.smtp.starttls.enable"] = mailUseTLS
+    }
```

Comment:
You can also do this
```suggestion
    private val props = Properties().apply {
        set("mail.smtp.host", mailHost)
        set("mail.smtp.port", mailPort)
        set("mail.smtp.auth", mailUseAuth)
        set("mail.smtp.starttls.enable", mailUseTLS)
    }
```
dunno which is better, you can choose